<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>910</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="configbn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>760</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendbn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>760</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>460</y>
     <width>229</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="sendlayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="smaclab">
      <property name="text">
       <string>发送mac</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="smacline"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="rmaclab">
      <property name="text">
       <string>接收mac</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="rmacline"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="length1lab">
      <property name="text">
       <string>数据包长度</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lengthline"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="directionlab">
      <property name="text">
       <string>配置方向</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="directionBox">
      <item>
       <property name="text">
        <string>全局配置</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tcam配置</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="configreqlab">
      <property name="text">
       <string>配置请求</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="configreqBox">
      <item>
       <property name="text">
        <string>配置请求有效</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>配置请求无效</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="configwrlab">
      <property name="text">
       <string>读写配置</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="configwrBox">
      <item>
       <property name="text">
        <string>读</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>写</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="configclrlab">
      <property name="text">
       <string>是否清零</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="configclrBox">
      <item>
       <property name="text">
        <string>是</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>否</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="configdatalab">
      <property name="text">
       <string>配置内容</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QTextEdit" name="configdatatext"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="globaladdrlab">
      <property name="text">
       <string>全局地址</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="globaladdrline"/>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>20</width>
     <height>661</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>291</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="sysreglayout">
    <item>
     <widget class="QLabel" name="sysreglabel">
      <property name="text">
       <string>系统状态</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="sysregtable"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>581</width>
     <height>881</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="portreglayout" stretch="0,0,0,0,0,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="port0label">
      <property name="text">
       <string>PORT0</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="port0layout">
      <item>
       <widget class="QLabel" name="init0label">
        <property name="text">
         <string>initiatior</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="init0table"/>
      </item>
      <item>
       <widget class="QLabel" name="taraget0label">
        <property name="text">
         <string>target</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="taraget0table"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="port1label">
      <property name="text">
       <string>PORT1</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="port1layout">
      <item>
       <widget class="QLabel" name="init1label">
        <property name="text">
         <string>initiatior</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="init1table"/>
      </item>
      <item>
       <widget class="QLabel" name="taraget1label">
        <property name="text">
         <string>target</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="taraget1table"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="port3label">
      <property name="text">
       <string>PORT3</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="port3layout">
      <item>
       <widget class="QLabel" name="init3label">
        <property name="text">
         <string>initiatior</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="init3table"/>
      </item>
      <item>
       <widget class="QLabel" name="taraget3label">
        <property name="text">
         <string>target</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="taraget3table"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="port4label">
      <property name="text">
       <string>PORT4</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="port4layout">
      <item>
       <widget class="QLabel" name="init4label">
        <property name="text">
         <string>initiatior</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="init4table"/>
      </item>
      <item>
       <widget class="QLabel" name="taraget4label">
        <property name="text">
         <string>target</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="taraget4table">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
